bundles/org.openhab.persistence.influxdb/README.md

```diff
@@ -34,6 +34,12 @@ This is very easy and you will find good documentation on it on the
 
 ## Configuration
 
+### Configuration via Web-Interface:
+All settings for this Persistance Service can be found under `Settings` → `Other Services` → `InfluxDB Persistance Service`
+
+Attention: The File based Configuration (described below) overrides the Configuration in the Web-Interface!
+
+### File based configuration (not needed when done via UI)
```

Comment:
```suggestion
### File-based configuration
```

I would then suggest to move the sentence " (not needed when done via UI)" after next line, i.e.:
```
This service can be configured in the file `services/influxdb.cfg`.
This is not needed when configured in UI.
```